case-study/ncqa.html

```diff
@@ -0,0 +1,52 @@
+
+  <figure class="left">
+    <img src="{{ site.baseurl }}/images/case-studies/ncqa/fig-1-goal-inventory.png" alt="Goal Inventory" width="400px">
+    <figcaption>Figure 1. Clinicians assist patients in completing various questionnaires. This is a sample Patient Recorded Outcome Measurement (PROM) question, as it is asked and recorded by a clinician.</figcaption>
+  </figure>
+  <br/>
+  <figure class="left">
+    <img src="{{ site.baseurl }}/images/case-studies/ncqa/fig-2-pain-scale.png" alt="Pain Scale" width="400px">
+    <figcaption>Figure 2. Following a patient-completed PROM, the clinician then views a patient’s results and answers several follow-up questions.</figcaption>
+  </figure>
+  <br/>
+  <figure class="left">
+    <img src="{{ site.baseurl }}/images/case-studies/ncqa/fig-3-prom-results.png" alt="PROM Results" width="400px">
+    <figcaption>Figure 3. Within the app a caregiver records the top three goals for a patient for the next specified period of time, ensuring a shared understanding between the patient and their clinicians.</figcaption>
+  </figure>
+
+  <p>Each time the patient has a care visit, they fill out one part of the survey; then the practitioner uses the second part of the survey to lead discussion and coordinate goals and focus areas for care. The second component of the survey tool uses a software development kit for computerized adaptive tests (CAT) <a href="https://www.assessmentcenter.net/">developed at Northwestern</a>; the app leverages adaptive technology to dramatically simplify the questionnaire flow.</p>
+  <p>The new survey tool meets a number of needs specific to the healthcare space; data protection and audit logs were crucial. Care providers need to be able to use the application when not actively connected to the internet, so the app saves its internal state and pushes updates to the server when next online. Privacy is also a large concern in a health care environment; all of the patient data is securely stored in a database with a robust permissioning system, to keep in line with HIPAA requirements. Enterprise deployment was another consideration for this application; organizations are able to manage deployment of the survey app across their fleet of devices through the Apple Enterprise Program.</p>
+</section>
+
+<section>
+  <h2>Benefits</h2>
+  <p>NCQA has seen a number of benefits come from the development and deployment of the platform. The survey tool makes it easier for practitioners to stay focused on their interactions with patients by streamlining the questionnaire process. Because all the data is entered digitally and stored in a centralized system, multiple caregivers can get access to the latest goals and outcomes for a patient, enhancing coordination of care especially in the case of complex, high-need patients.
+  NCQA was pleased with both the end product and the process of working with SmartLogic.</p>
+  <p>SmartLogic really knows how to get customers to an end product.
```

Comment:
@rozerosie should there be quotation marks around this and the next quote two lines down? if so can you please add them? i reviewed the rest and it looks good. so pending your decision of whether to add quotation marks just merge this PR (don't wait for me) and then delete the branch.